WordPress/src/main/java/org/wordpress/android/ui/accounts/SignInFragment.java

```diff
@@ -148,7 +148,7 @@ public void onCreate(Bundle savedInstanceState) {
             mSelfHosted = savedInstanceState.getBoolean(KEY_IS_SELF_HOSTED);
         }
 
-        mInhibitMagicLogin = getActivity() != null
+        mShouldReturnToPreviousActivity = getActivity() != null
```

Comment:
If we go with the rename, let's add some comment here to explain for example why an email client being unavailable governs the return to the previous activity or not. Thanks!